async_work_promise/napi/binding.c

```diff
@@ -0,0 +1,183 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define NAPI_EXPERIMENTAL
```

Comment:
Nit: is this line redundant ?

Reply:
I guess it's unnecessary if ever N-API used is already stable.